internal/mcpprobe/refused_test.go

```
@@ -0,0 +1,44 @@
+package mcpprobe
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/modelcontextprotocol/go-sdk/jsonrpc"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(goimports)